docs/components/receive.md

```diff
@@ -66,6 +74,13 @@ The example content of `hashring.json`:
 With such configuration any receive listens for remote write on `<ip>10908/api/v1/receive` and will forward to correct one in hashring if needed
 for tenancy and replication.
 
+### Enabling Routing/Ingestion for Thanos Receiver
+
+We have not added yet another flag for controlling this behaviour. We have relied on the below convention to enable ingestion/routing -
+* **Ingestion and Distribution** - Make sure `--receive.local-endpoint` and `--receive.hashrings` / `--receive.hashrings-file` are provided.
+* **Ingestion** - Since routing behaviour is not needed, just provide `--receive.local-endpoint` only.
```

Comment:
Let's avoid words like "just", "simply," and "obviously" in general since they can imply that things are easy, which may not be the case for every user. In this case we are using "just" as a synonym for "only" but we we already say "only" at the end, so we don't need to say it twice IMO.
```suggestion
* **Ingestion** - Since routing behaviour is not needed, provide `--receive.local-endpoint` only.
```